lib/realtime/messages.ex

```diff
@@ -14,7 +14,7 @@ defmodule Realtime.Messages do
   def create_message(attrs, conn, opts \\ [mode: :savepoint]) do
     channel = Message.changeset(%Message{}, attrs)
 
-    result =
+    {:ok, result} =
```

Comment:
@filipecabaco do you intend for this to raise do to left not matching right?

otherwise, should stick with `result = ...` for this and then update the `case result do...` below:

```elixir
case result do
  %Ecto.Changeset{valid?: false} = error -> {:error, error}
  {:error, error} -> {:error, error}
  {:ok, result} -> {:ok, result}
end
```